cli/azd/internal/repository/initializer.go

```diff
@@ -147,21 +168,29 @@ func (i *Initializer) promptForDuplicates(ctx context.Context, staging string, t
 			i.console.Message(ctx, fmt.Sprintf(" * %s", file))
 		}
 
-		overwrite, err := i.console.Confirm(ctx, input.ConsoleOptions{
-			Message:      "Overwrite files with versions from template?",
-			DefaultValue: false,
-		})
-
-		if err != nil {
-			return fmt.Errorf("prompting to overwrite: %w", err)
-		}
-
-		if !overwrite {
-			return errors.New("confirmation declined")
+		if selection, err := i.console.Select(ctx, input.ConsoleOptions{
+			Message: "What would you like to do with these files?",
+			Options: []string{
+				"Overwrite with versions from template",
+				"Keep the current files as-is",
+			},
+		}); err != nil {
+			return nil, fmt.Errorf("prompting to overwrite: %w", err)
+		} else {
+			switch selection {
+			case 0:
+				return nil, nil
+			case 1:
+				skipSourceFiles = make([]string, len(duplicateFiles))
+				for i, file := range duplicateFiles {
+					skipSourceFiles[i] = filepath.Join(staging, file)
+				}
+				return skipSourceFiles, nil
+			}
```

Comment:
nit,
consider using the form:
```
x,e := func()
if e != nil {
   return
}
switch x {
  ....
}
```

Mainly b/c the condition is not within the same line as the `if`, and secondly to save one level of indentation.

Reply:
There was something about the `switch selection` block being close to where `console.Select` was that made the code look more logical to me. I've changed it to the form as suggested, since it doesn't really matter that much.